src/v/features/feature_table.cc

```diff
@@ -103,6 +103,8 @@ std::string_view to_string_view(feature f) {
         return "cluster_topic_manifest_format_v2";
     case feature::shard_placement_persistence:
         return "shard_placement_persistence";
+    case feature::unified_tx_state:
+        return "tx_log_state_revamp";
```

Comment:
nit: missing rename

Reply:
😮‍💨 fixed, I had to rebase because of conflict in this file.